setup.py

```diff
@@ -108,6 +108,7 @@ def get_git_sha() -> str:
         "pandas[performance]>=2.0.3, <2.1",
         "parsedatetime",
         "pgsanity",
+        "playwright",
```

Comment:
Should we move this into an optional dep until selenium is deprecated?

Reply:
Where are the optional deps located?

Reply:
That's a good idea. @kgabryje in `extras_require` below.

Reply:
Thanks, moved